### ISSUE TITLE
No upgrade path from v1alpha2

### ISSUE THREAD

**Issue by jrgirvan** [GoogleContainerTools/kpt#2386]
### Expected behavior

`kpt fn eval -i gcr.io/kpt-fn/fix:v0.2` works on `v1alpha2` Kpt files

One of these would be acceptable:
- works on `v1alpha2` Kptfile's
- Errors with a message other that running the same command
- Inform the user upgrades from `v1alpha2` aren't supported
- nothing as this required the user to have upgraded their package with an alpha version of kpt

### Actual behavior

```bash
❯ kpt fn eval -i gcr.io/kpt-fn/fix:v0.2
Error: Kptfile at "/home/me/src/kpt-package/my/pkg" has an old version ("v1alpha2") of the Kptfile schema.
Please run "kpt fn eval <PKG_PATH> -i gcr.io/kpt-fn/fix:v0.2" to upgrade the package and retry. 
```

### Information

```bash
❯ kpt version
1.0.0-beta.1
```

### Steps to reproduce the behavior

Run `kpt fn eval -i gcr.io/kpt-fn/fix:v0.2` on any package with a Kptfile of version `v1alpha2`

**Comment by mikebz** [comment]
@phanimarupaka yours for commentary

**Comment by phanimarupaka** [comment]
@jrgirvan Thanks for filing the issue. The issue is the with the error message which we have fixed and will release it soon.

You should include the flag `--include-meta-resources`, so the command becomes.

```
kpt fn eval -i gcr.io/kpt-fn/fix:v0.2 --include-meta-resources
```

**Comment by jrgirvan** [comment]
Awesome thanks @phanimarupaka
